From d9b3a28a579ba1969e6fa3cb83d487f7742e6765 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 30 Jul 2015 17:25:59 +0200
Subject: [PATCH 1/2] theme: Make menu ornament slightly wider

The existing width works well enough for a narrow character like the dot,
but doesn't leave any whitespace for a wider one like the checkmark.

https://bugzilla.gnome.org/show_bug.cgi?id=741366
---
 data/theme/gnome-shell.css | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/theme/gnome-shell.css b/data/theme/gnome-shell.css
index 80dff57..97634b1 100644
--- a/data/theme/gnome-shell.css
+++ b/data/theme/gnome-shell.css
@@ -140,7 +140,7 @@ StScrollBar StButton#vhandle:active {
 
 .popup-menu-ornament {
     text-align: right;
-    width: 1em;
+    width: 1.2em;
 }
 
 .popup-menu-boxpointer,
-- 
2.4.3


From 1e5fc64944e9259782d060032e48bb7e6aaaabd3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 30 Jul 2015 15:54:16 +0200
Subject: [PATCH 2/2] windowMenu: Use CHECK ornament rather than DOT

The dot suggests a radio action, while the items are actually toggles;
in the app menu we use checkmarks in that case, so do the same here.
---
 js/ui/windowMenu.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/js/ui/windowMenu.js b/js/ui/windowMenu.js
index 3d378fa..43f6125 100644
--- a/js/ui/windowMenu.js
+++ b/js/ui/windowMenu.js
@@ -74,7 +74,7 @@ const WindowMenu = new Lang.Class({
                 window.make_above();
         }));
         if (window.is_above())
-            item.setOrnament(PopupMenu.Ornament.DOT);
+            item.setOrnament(PopupMenu.Ornament.CHECK);
         if (window.get_maximized() == Meta.MaximizeFlags.BOTH ||
             type == Meta.WindowType.DOCK ||
             type == Meta.WindowType.DESKTOP ||
@@ -93,7 +93,7 @@ const WindowMenu = new Lang.Class({
                     window.stick();
             }));
             if (isSticky)
-                item.setOrnament(PopupMenu.Ornament.DOT);
+                item.setOrnament(PopupMenu.Ornament.CHECK);
             if (window.is_always_on_all_workspaces())
                 item.setSensitive(false);
 
-- 
2.4.3

